content/en/cosign/overview.md

```diff
@@ -15,98 +15,84 @@ features:
 
 ![Cosign Overview](/sigstore_cosign-horizontal-color.svg)
 
-Cosign supports container signing, verification, and storage in an OCI registry.
-Cosign aims to make signatures invisible infrastructure.
-
-![Cosign demo gif](/cosign.gif)
-
-Cosign supports:
+## Getting Started (Quick Start)
 
-<list :items="features" type="info"></list>
+Cosign supports blob and container signing, verification, and storage in an OCI registry.
+Cosign aims to make signatures invisible infrastructure.
```

Comment:
```suggestion
Cosign aims to make signatures into invisible infrastructure.
```

Reply:
Added "into" because the sentence could be confusing without. But does this sentence belong here, actually? Is the point about invisible infrastructure related to the quickstart information about signing, verification, and storage, or is it a more general descriptive point that should go elsewhere?

Reply:
I will remove it.   It can go on the main concepts page.